diff --git a/index.js b/index.js
--- a/index.js
+++ b/index.js
@@ -150,7 +150,7 @@ const plugin = function (defaults = {}) {
 				}
 
 				// then, create a group for each config
-				const id = config.split ? `${hash}-${groups.size}` : hash;
+				const id = false ? `${hash}-${groups.size}` : hash;
 				let group = groups.get(id);
 				if (!group) {
 					group = { i: groups.size, config, routes: [] };
